docs/source/javascripts/site.js

```diff
@@ -60,6 +60,17 @@ window.modules.search = (function () {
             return item;
         },
 
+        pluginResultListItemTPL = function (name, git_hub_key) {
+            var item = '';
+            item += '<p class="search-results__title"> Looking for the ' + name + " plugin? - ";
```

Comment:
Shouldn't be linking them to https://plugins.workarea.com rather than GitHub? This would prevent searching for proprietary plugins.

Reply:
I dont think plugins isnt building anymore, so github seemed like the save option.

Reply:
I'm not downvoting your solution, I'm downvoting the fact that plugins no longer builds READMEs...